.travis.yml

```diff
@@ -9,20 +9,24 @@ git:
 os:
   - linux
 env:
-  - V=0.28.0
+  # TRAVIS_TAG should be set as "<artifactSuffix>--<version>"; otherwise publishing will abort
+  global:
+    - BAZEL=0.28.0
+    - COMPILER_CLI_ARTIFACT_ID="play-routes-compiler-cli_$(awk -F '--' '{print $1}' <<< $TRAVIS_TAG)"
+    - COMPILER_CLI_VERSION="$(awk -F '--' '{print $2}' <<< $TRAVIS_TAG)"
 before_install:
   # Borrowed from rules_scala
   - |
     sysctl kernel.unprivileged_userns_clone=1
     sudo apt-get update -q
     sudo apt-get install libxml2-utils -y
     OS=linux
-    if [[ $V =~ .*rc[0-9]+.* ]]; then
```

Comment:
Very minor: What's the reason changing this variable name?

Reply:
`V` isn't descriptive